### DISCUSSION TITLE
I want to create projects using npx creact-next-app@11 but the next version created is still the latest version and how do I need to do it

### DISCUSSION THREAD

**Discussion by sgzweb-new** [vercel/next.js#65782] [Help]
### Summary

I want to create projects using npx creact-next-app@11 but the next version created is still the latest version and how do I need to do it，please help me

**Comment by icyJoseph** [comment]
Create the project, and then run `npm i next@11`, or whichever applies to your package manager setup. Make sure to remove `node_modules` and double check that peer dependencies are still correctly working.
